pkgs/applications/misc/rofi-rbw/default.nix

```
@@ -0,0 +1,25 @@
+{ lib, buildPythonApplication, fetchFromGitHub, configargparse, }:
+
+buildPythonApplication rec {
+  pname = "rofi-rbw";
+  version = "0.5.0";
+  format = "pyproject";
+
+  src = fetchFromGitHub {
+    owner = "fdw";
+    repo = "rofi-rbw";
+    rev = version;
+    hash = "sha256-1RDwb8lKls6+X/XtARbi4F7sK4nT03Iy3Wb9N1LEa5o=";
+  };
+
+  pythonImportsCheck = [ "rofi_rbw" ];
+  propagatedBuildInputs = [ configargparse ];
```

```suggestion
  propagatedBuildInputs = [ configargparse ];
  
  pythonImportsCheck = [ "rofi_rbw" ];  
```
To keep some kind of cleanliness into the code, the build process specs should go first, and the checks should go last.